packages/antd/src/editable/index.tsx

```diff
@@ -162,7 +162,7 @@ Editable.Popover = observer((props) => {
       className={cls(prefixCls, props.className)}
       content={props.children}
       trigger="click"
-      destroyTooltipOnHide
+      // destroyTooltipOnHide
```

Comment:
if destory, this error will happen. 
![image](https://github.com/alibaba/formily/assets/18055018/da826d58-f510-4d57-805d-a0f1f10fb13b)

Reply:
fixed.